packages/core/src/auth/sso/clients.ts

```diff
@@ -33,6 +33,12 @@ import { DevSettings } from '../../shared/settings'
 import { SdkError } from '@aws-sdk/types'
 import { HttpRequest, HttpResponse } from '@aws-sdk/protocol-http'
 import { StandardRetryStrategy, defaultRetryDecider } from '@aws-sdk/middleware-retry'
+import OidcClientPKCE from './oidcclientpkce'
+import { toSnakeCase } from '../../shared/utilities/textUtilities'
+import { Credentials, Service } from 'aws-sdk'
+import apiConfig = require('./service-2.json')
```

Comment:
I see we import package.json like the following, maybe this works:
`import packageJson from '../../package.json'`

Reply:
Yeah that should work as well, I was mostly just copying how other teams imported their json files for apiConfig